src/css/var/isAutoPx.js

```diff
@@ -0,0 +1,36 @@
+define( function() {
+
+"use strict";
+
+return function( prop ) {
+
+	// The second regex visualized:
+	//
+	//                                       /----------\
+	//                   /-------------\    |            |
+	//                  |               |   |  / Top  \  |
+	//                  |  / Margin  \  |   | | Right  | |
+	//         /--------+-|           |-+---+-| Bottom |-+---\
+	//        |            \ Padding /         \ Left /       |
+	//        |                                               |
+	//        |                /---------\                    |
+	//        |               |           |                   |- END
+	//        |               |  / Min \  |    / Width  \     |
+	//        |---------------+-|       |-+---|          |----|
+	//        |                  \ Max /       \ Height /     |
+	// BEGIN -|                                               |
+	//        |                /----------\                   |
+	//        |               |            |    /-------\     |
+	//        |               |  / Top  \  |   |         |    |
+	//         \--- Border ---+-| Right  |-+---+- Width -+---/
+	//                          | Bottom |
+	//                           \ Left /
+
+	// The first test is used to ensure that:
+	// 1. The prop starts with a lowercase letter (as we uppercase it for the second regex).
+	// 2. The prop is not empty.
+	return /^[a-z]$/.test( prop[ 0 ] ) &&
```

Comment:
Having a function is fine, but we should continue with the convention of creating regexes once outside of functions.